pyqg/model.py

```diff
@@ -550,6 +552,24 @@ def _step_forward(self):
 
         # the basic steps are
         self._print_status()
+    
+    def _invert(self):
+        self.kernel.invert()
+
+    def _do_advection(self):
+        self.kernel.do_advection()
+
+    def _do_friction(self):
+        self.kernel.do_friction()
+
+    def _do_q_subgrid_parameterization(self):
+        self.kernel.do_q_subgrid_parameterization()
+
+    def _do_uv_subgrid_parameterization(self):
+        self.kernel.do_uv_subgrid_parameterization()
+
+    def _forward_timestep(self):
+        self.kernel.forward_timestep()
```

Comment:
This feels like a code smell. Can we avoid wrapping all these methods? Could we use delegate here?